src/main/java/frc/robot/subsystems/intake/IntakeIOSim.java

```diff
@@ -0,0 +1,50 @@
+package frc.robot.subsystems.intake;
+
+import com.ctre.phoenix6.controls.PositionVoltage;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.units.Angle;
+import edu.wpi.first.units.MutableMeasure;
+import edu.wpi.first.units.Units;
+import edu.wpi.first.units.Velocity;
+import lib.motors.SparkMaxSim;
+import lib.motors.TalonFXSim;
+
+public class IntakeIOSim implements IntakeIO {
+    private TalonFXSim angleMotor;
+    private SparkMaxSim frontRoller;
+    private SparkMaxSim centerRoller;
+    private PositionVoltage positionRequest = new PositionVoltage(0);
```

Comment:
```suggestion
    private final TalonFXSim angleMotor;
    private final SparkMaxSim frontRoller;
    private final SparkMaxSim centerRoller;
    private final PositionVoltage positionRequest = new PositionVoltage(0);
```

Reply:
done. 
also we already have a method for convenient retracting called setAngle() in Intake class

Reply:
`setAngle` is not a convenience method, it's a method used for all angle settings. What I meant is that e.g the open method will not receive parameters but use a constant value.

Reply:
423a178e32ada435b677b785a6c731fd229beeee

Reply:
And what about `setAngle`?